.drone/drone.jsonnet

```diff
@@ -266,6 +266,41 @@ local deploy_to_dev() = {
               for d in aws_serverless_deployments
             ],
   },
+  // Build and deploy serverless code packages
```

Comment:
is there a reason to move this to drone? it was working fine in GHA.

Reply:
That's my bad. I think I just saw jsonnet and became unnecessarily enraged. Let me give this a real review.

Reply:
Sorry for enraging you 😃. I might change my mind on Drone before this is merged (I believe it would be an even harder sell for the mimir team, they don't even have a config yet). To be continued next week

Reply:
I put it back in Github Actions. Least friction for you, indeed. But it is unbearably slow. 90 secs in Drone vs 13 minutes in Github actions

Reply:
Done!